phys2bids/tests/test_phys2bids.py

```diff
@@ -41,3 +42,23 @@ def test_print_json(tmpdir):
         loaded_data = json.load(src)
 
     assert test_json_data == loaded_data
+
+
+def test_raise_exception(samefreq_full_acq_file):
+    test_filename = 'input.txt'
+
+    with raises(Exception) as errorinfo:
+        phys2bids.phys2bids(filename=test_filename,chtrig=0)
+    assert 'Wrong trigger' in str(errorinfo.value)
+
+    test_path, test_filename = os.path.split(samefreq_full_acq_file)
+    with raises(Exception) as errorinfo:
+        phys2bids.phys2bids(filename=test_filename, num_timepoints_expected=[70], tr=[1.3, 2],
+                            indir=test_path, outdir=test_path)
+    assert 'doesn\'t match' in str(errorinfo.value)
+
```

Comment:
what is the propouse of this?

Reply:
Could you clarify what you mean? Are you talking about the backslash?

Reply:
No, no, I wasn't properly understanding how errorinfo.value worked. My apologies